### PR TITLE
fix: update dependencies

### PR DESCRIPTION
**Pull request by benoitdeziel**
## Description

This PR updates the dependencies of all projects. It also set the peerDependencies.

## Checklist

- [x] Updated the version number in the `package.json`
- [x] Test any new configuration on a repo using [`npm link`](https://docs.npmjs.com/cli/link)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,17 +1,23 @@
...
-    "prettier": "^1.18.2",
-    "prettier-config-landr": "^0.0.3",
-    "typescript": "^3.5.2"
-  }
+    "name": "linting-and-formatting",
+    "repository": "git@github.com:Mixgenius/linting-and-formatting.git",
+    "author": "LANDR",
+    "license": "MIT",
+    "private": true,
+    "workspaces": [
```
Comment (robertcoopercode) [inline package.json]: Do we have to run special commands when using yarn workspaces? If yes, can you add some documentation on the correct commands to run?

Comment (benoitdeziel) [reply package.json]: Nope, just run `yarn`, it will install everything

Comment (robertcoopercode) [reply package.json]: Ok, cool

Comment (davidmartinezmarin) [reply package.json]: @benoitdeziel -> nothing on the readme about that ?

Comment (benoitdeziel) [reply package.json]: I could make another PR for that if you want

**Pull request merged**